frontend/controller/service-worker.js

```diff
@@ -104,9 +104,16 @@ sbp('sbp/selectors/register', {
               break
             }
             case 'navigate': {
+              if (data.groupID) {
+                sbp('state/vuex/commit', 'setCurrentGroupId', { contractID: data.groupID })
+              }
               sbp('controller/router').push({ path: data.path }).catch(console.warn)
               break
             }
+            case 'sbp': {
+              sbp(...deserializer(event.data.data))
+              break
+            }
```

Comment:
Can you add a comment here explaining what this addition is for?


Also, a reminder to update `push.js` to address:

```js
      if (data.contractType === 'gi.contracts/chatroom') {
        // TODO: Text for this notification
        return makeNotification({ title: '@@@err', body: e.message })
      }
```

Reply:
> Can you add a comment here explaining what this addition is for?

What do you mean by that? What needs to be documented here? I think that what it does is pretty obvious (execute SBP invocations from the SW to the browser). The purpose is perhaps less clear (it's currently used by the 'notificationclick' handler) for notifications that have an 'sbpInvocation' instead of a path.

Can you clarify what needs to be documented and why this `case` is special over all the others here?

Reply:
We need to explain why it's here, what it's used for.

> The purpose is perhaps less clear (it's currently used by the 'notificationclick' handler) for notifications that have an 'sbpInvocation' instead of a path.

You can just copy/paste this. 👆